index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Desafio dos diamantes</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <select name="" id="select_diamante">
+        <option value="" selected>Selecione uma opção</option>
+    </select>
+
+    <div class="mostra_diamante" id="mostra_diamante"></div>
```

Comment:
```suggestion
    <div class="mostra-diamante" id="mostra_diamante"></div>
```

Classes no front-end deveriam possuir hífen ao invés de underline.

Reply:
alterado